### PR TITLE
chore: add more makefile output prompts

### PR DESCRIPTION
**Pull request by leslie-tsang**
1. add more makefile target output prompts
2. fix target `help` prompts to sync with linux runtime

> before
```bash
docker build output ...
```

> after
```bash
[ info ] build-on-centos -> [ Start  ]
docker build output ...
[ info ] build-on-centos -> [ Done  ]
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,86 +63,110 @@ endef
 ### build-on-centos : Build apache/apisix:xx-centos image
 .PHONY: build-on-centos
 build-on-centos:
+	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(ENV_DOCKER) build -t $(ENV_APISIX_IMAGE_TAG_NAME)-centos -f ./centos/Dockerfile .
+	@$(call func_echo_success_status, "$@ -> [ Done  ]")
```
Comment (tokers) [inline Makefile:68]:
```suggestion
	@$(call func_echo_success_status, "$@ -> [ Done ]")
```

File: Makefile [hunk 2]
```diff
@@ -63,86 +63,110 @@ endef
...
 
 
 ### build-on-alpine : Build apache/apisix:xx-alpine image
 .PHONY: build-on-alpine
 build-on-alpine:
+	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(ENV_DOCKER) build -t $(ENV_APISIX_IMAGE_TAG_NAME)-alpine -f ./alpine/Dockerfile .
+	@$(call func_echo_success_status, "$@ -> [ Done  ]")
```
Comment (tokers) [inline Makefile:76]:
```suggestion
	@$(call func_echo_success_status, "$@ -> [ Done ]")
```

File: Makefile [hunk 3]
```diff
@@ -63,86 +63,110 @@ endef
...
 
 
 ### build-on-alpine-local : Build apache/apisix:xx-alpine-local image
 # Actually it is not build on certain version but on local code
 # Use this name (in the same patterns with others) for convenient CI
 .PHONY: build-on-alpine-local
 build-on-alpine-local:
+	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(ENV_DOCKER) build -t $(ENV_APISIX_IMAGE_TAG_NAME)-alpine-local --build-arg APISIX_PATH=${APISIX_PATH} -f ./alpine-local/Dockerfile .
+	@$(call func_echo_success_status, "$@ -> [ Done  ]")
```
Comment (tokers) [inline Makefile:86]:
```suggestion
	@$(call func_echo_success_status, "$@ -> [ Done ]")
```

File: Makefile [hunk 4]
```diff
@@ -63,86 +63,110 @@ endef
...
 
 ### push-on-centos : Push apache/apisix:xx-centos image
 # centos not support multiarch since it reply on x86 rpm package
 .PHONY: push-on-centos
 push-on-centos:
+	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(ENV_DOCKER) push $(ENV_APISIX_IMAGE_TAG_NAME)-centos
 	$(ENV_DOCKER) build -t $(IMAGE_NAME):latest -f ./centos/Dockerfile .
 	$(ENV_DOCKER) push $(IMAGE_NAME):latest
+	@$(call func_echo_success_status, "$@ -> [ Done  ]")
```
Comment (tokers) [inline Makefile:97]:
```suggestion
	@$(call func_echo_success_status, "$@ -> [ Done ]")
```

File: Makefile [hunk 5]
```diff
@@ -63,86 +63,110 @@ endef
...
 
 ### push-on-alpine : Push apache/apisix:xx-alpine image
 .PHONY: push-multiarch-on-alpine
 push-multiarch-on-alpine:
+	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(ENV_DOCKER) buildx build --push \
 		-t $(ENV_APISIX_IMAGE_TAG_NAME)-alpine \
 		--platform linux/amd64,linux/arm64 \
 		-f ./alpine/Dockerfile .
+	@$(call func_echo_success_status, "$@ -> [ Done  ]")
```
Comment (tokers) [inline Makefile:108]:
```suggestion
	@$(call func_echo_success_status, "$@ -> [ Done ]")
```

File: Makefile [hunk 6]
```diff
@@ -63,86 +63,110 @@ endef
...
 
 
 ### build-on-alpine-cn : Build apache/apisix:xx-alpine image (for chinese)
 .PHONY: build-on-alpine-cn
 build-on-alpine-cn:
-	$(ENV_DOCKER) build -t $(IMAGE_NAME):${APISIX_VERSION}-alpine --build-arg APISIX_VERSION=${APISIX_VERSION} --build-arg ENABLE_PROXY=true -f alpine/Dockerfile alpine
+	@$(call func_echo_status, "$@ -> [ Start ]")
+	$(ENV_DOCKER) build -t $(ENV_APISIX_IMAGE_TAG_NAME)-alpine --build-arg APISIX_VERSION=$(APISIX_VERSION) --build-arg ENABLE_PROXY=true -f alpine/Dockerfile alpine
+	@$(call func_echo_success_status, "$@ -> [ Done  ]")
```
Comment (tokers) [inline Makefile:116]: DItto.

Comment (leslie-tsang) [reply Makefile]: @tokers fix it.

**Review by gxthrj — APPROVED** [review]
LGTM

**Review by bzp2010 — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**